stereotype/model.py

```diff
@@ -148,6 +148,16 @@ def __getitem__(self, key):
             raise KeyError(key)  # Avoids getting methods via getitem
         return getattr(self, key)
 
+    def get(self, key, default=None):
+        """
+        Returns a value of a field with the given key, or the provided default if no such field exists
+        or a required field is missing value.
+        """
+        if key not in self.__slots__:
```

Comment:
BTW I just realized it might be a good idea to use a set for this check (constructed by the metaclass) instead of the usual list/tuple. But it's a preexisting issue, I'll implement it tomorrow with some bug fix.